frontend/src/store/ActionModal/index.js

```diff
@@ -0,0 +1,53 @@
+import { action, makeObservable, observable } from 'mobx'
+
+export const ActionModalName = {
+  STREANMING: 'streaming',
+  SAVE_ROBOT_SETTINGS: 'saveRobotSettings'
+}
+
+class ActionModal {
+  constructor() {
+    makeObservable(this)
+  }
+
+  @observable actionModalName = null
+  @observable showActionModal = false
+  @observable actionModalConfirmCallback = null
+  @observable actionModalCancelCallback = null
+
+  @action
+  onShowActionModal  = ({ actionModalName, actionModalConfirmCallback, actionModalCancelCallback }) => {
+    this.actionModalName = actionModalName
+    this.showActionModal = true
+    if(typeof actionModalConfirmCallback === 'function') {
```

Comment:
seems like a missing formatting within this file

Reply:
done